### ISSUE TITLE
Unable to edit Alt text

### ISSUE THREAD

**Issue by nibbles-** [pachli/pachli-android#1381]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots or video**
If applicable, please add screenshots or video to help explain your problem.

**Links**
If a particular post or posts demonstrate the problem please link to them here.

**Versions**
Please include the version of Pachli and Android you are using, and the software version your server is using. Find this on the Pachli "About" screen.

**Additional context**

**Affirmation**
I have checked other issues for this project (open and closed) and I cannot find one that matches the problem I am reporting.
